### PR TITLE
Add Python Solution for "Censoring (Bronze)" in "Simulation" Module

### PR DESCRIPTION
**Pull request by aaguy-hue**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

File: solutions/usaco-526.mdx [hunk 1]
```diff
@@ -13,6 +13,30 @@ author: Brad Ma
...
+	final = ""
+	# Add each character to the final string
+	for char in s:
+		final += char
+		# If the last len(t) characters is just t, then remove the last len(t) characters
+		if len(final) >= len(t) and final[-len(t):] == t:
+			final = final[:-len(t)]
+	return final
+
+print(main())
```
Comment (SansPapyrus683) [inline solutions/usaco-526.mdx:35]: take this out of main (unless it TLEs without)

File: solutions/usaco-526.mdx [hunk 2]
```diff
@@ -13,6 +13,30 @@ author: Brad Ma
...
+sys.stdout = open("censor.out", "w")
+
+def main():
+	s = input().strip()
+	t = input().strip()
+	final = ""
+	# Add each character to the final string
+	for char in s:
+		final += char
+		# If the last len(t) characters is just t, then remove the last len(t) characters
```
Comment (SansPapyrus683) [inline solutions/usaco-526.mdx:30]: this is around 80 chars?

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-526.mdx (hunk 1 above)
Comment (aaguy-hue) [reply solutions/usaco-526.mdx]: Okay, I made that change.

File: solutions/usaco-526.mdx (hunk 2 above)
Comment (aaguy-hue) [reply solutions/usaco-526.mdx]: Sorry, I don't quite understand what you mean. What I meant was that if the new string ends with t, we remove t from the end. I edited this comment in a new commit, so hopefully it makes more sense.

Comment (SansPapyrus683) [reply solutions/usaco-526.mdx]: lines should be like under 80 chars
```py
"""
to split a comment use a multiline
string like this
"""
```

Comment (aaguy-hue) [reply solutions/usaco-526.mdx]: Oh, I understand what you mean now. I fixed this in commit 16ed191.

**Review by SansPapyrus683 — APPROVED** [review]
we chillin

**Pull request merged**